spec/views/helpers/custom_links_helper_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+require 'rails_helper'
+
+RSpec.describe CustomLinksHelper, :type => :helper do
```

Comment:
__[rubocop]__
* Style/HashSyntax: Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)

<!-- Comment by [SideCI](https://sideci.com) -->